setup/webmail.sh

```diff
@@ -157,11 +189,16 @@ usermod -a -G dovecot www-data
 # could use dovecot instead of www-data, but not sure it matters
 chown root.www-data $STORAGE_ROOT/mail
 chmod 775 $STORAGE_ROOT/mail
-chown root.www-data $STORAGE_ROOT/mail/users.sqlite 
-chmod 664 $STORAGE_ROOT/mail/users.sqlite 
+chown root.www-data $STORAGE_ROOT/mail/users.sqlite
+chmod 664 $STORAGE_ROOT/mail/users.sqlite
+
+# Fix Carddav permissions
+chmod -R 644 ${RCM_PLUGIN_DIR}/carddav
+chmod -R a+X ${RCM_PLUGIN_DIR}/carddav
```

Comment:
These two lines can be merged, I think, and since +X is pretty rare we should document what's going on there. But, also, why do we need this? Is this recommended by the carddav docs or do we not know what subdirectories are in there?